### ISSUE TITLE
Finalize "variable part" of metadataschema for variable report and variable detail page 

### ISSUE THREAD

**Issue by kpstephan** [dzhw/metadatamanagement#154]
this is a subset of issue #126 which should be finished first. Include all variable properties, which are relevant for Abs2005, Panel2008 and SE19 datasets

**Comment by rreitmann** [comment]
Duplicate of #126
